runtime/src/main/resources/assets/roughlyenoughitems/lang/uk_ua.json

```
@@ -1,43 +1,47 @@
 {
-  "text.rei.cheating": "Використання читів",
-  "text.rei.cheating_disabled": "§7Використання читів вимкнено",
-  "text.rei.cheating_enabled": "§cВикористання читів увімкнено",
-  "text.rei.cheating_limited_enabled": "§BВикористання читів увімкнено (Використання команд)",
-  "text.rei.cheating_enabled_no_perms": "§7Використання читів §cувімкнено §7(Без дозволу)",
-  "text.rei.cheating_limited_creative_enabled": "§aВикористання читів увімкнено (Використання творчого режиму)",
-  "text.rei.no_permission_cheat": "Для використання читів потрібні права оператора",
-  "text.rei.search.field.suggestion": "Пошук...",
+  "text.rei.cheating": "Шахрайство",
+  "text.rei.cheating_disabled": "§7Шахрайство вимкнено",
+  "text.rei.cheating_enabled": "§cШахрайство ввімкнено",
+  "text.rei.cheating_limited_enabled": "§bШахрайство ввімкнено (Команди)",
+  "text.rei.cheating_enabled_no_perms": "§7Шахрайство §cввімкнено §7(Без дозволу)",
+  "text.rei.cheating_limited_creative_enabled": "§aШахрайство ввімкнено (Творчість)",
+  "text.rei.no_permission_cheat": "Для шахрайста потрібні права оператора",
+  "text.rei.search.field.suggestion": "Пошук…",
   "text.rei.feedback": "Хочете залишити відгук розробнику REI? %s, щоб надіслати свій відгук!",
-  "text.rei.feedback.link": "Натисніть на мене, щоб відвідати Google Форму",
+  "text.rei.feedback.link": "Натисніть на мене, щоб відвідати Google-форму",
   "text.rei.support.me": "Підтримайте мене, щоб продовжувати покращувати REI!",
-  "text.rei.support.me.desc": "Як patreon, який підтримує REI, ви можете використовувати останню версію REI для снапшотів Minecraft.\nКрім того, ви також можете підтримати мене фінансово, орендуючи сервер BisectHosting.\n\n%s\n%s",
+  "text.rei.support.me.desc": "Як patreon, який підтримує REI, ви можете використовувати останню версію REI для знімків Minecraft.\nКрім того, ви також можете підтримати мене фінансово, орендуючи сервер BisectHosting.\n\n%s\n%s",
   "text.rei.support.me.patreon": "Підтримати через Patreon",
   "text.rei.support.me.bisect": "Скористайтеся кодом SHEDANIEL на BisectHosting",
   "text.rei.not.fully.initialized": "REI ще не повністю ініціалізовано!",
-  "text.rei.not.fully.initialized.tooltip": "Відсутні етапи: %s\nЯкщо це не зникне,\nспробуйте пошукати підтримку із логів!",
+  "text.rei.not.fully.initialized.tooltip": "Відсутні етапи: %s\nЯкщо це не зникне,\nспробуйте пошукати підтримку із журналів!",
   "text.rei.inventory.highlighting.enabled": "Підсвічування інвентарю увімкнено",
-  "text.rei.inventory.highlighting.enabled.tooltip": "Це робить слоти, які не збігаються\nз фільтром пошуку сірим.\nДвічі клацніть на панелі пошуку, щоб перемкнути цей режим.",
-  "text.rei.caching.search": "REI кешує результати пошуку...",
+  "text.rei.inventory.highlighting.enabled.tooltip": "Робить слоти, які не відповідають\nфільтру пошуку сірим.\nДвічі натисніть на панелі пошуку, щоб перемкнути режим.",
```

How about? I feel this reads more native.
```suggestion
  "text.rei.inventory.highlighting.enabled.tooltip": "Позначити слоти, які не відповідають\nфільтру пошуку сірим.\nДвічі натисніть на панелі пошуку, щоб перемкнути режим.",
```